06 Data Library/01 Forex/Timezone.html

```diff
@@ -0,0 +1,5 @@
+<p>
+OANDA data is in GMT timezone. Although many popular charting websites convert this to EST for display QuantConnect has elected to leave it in GMT and not manipulate the data. When comparing the data to external references make sure to account for the different time zones.
+
+While Oanda data is in GMT Time, the exchange is set to its local time, New York Time. So data accessed from this brokerage is timestamped in New York Time. Meanwhile, FXCM data is set in UTC-05 Time, or 5 hours behind UTC Time. It is observed in the Eastern Time Zone (New York Time) during standard time, and in the Central Time Zone (Chicago Time) during the other eight months
```

Comment:
Eastern Standard Time (EST) by definition is UTC-05, whereas Eastern Daylight Time (EDT) is UTC-04. New York Time is the amalgamation of the two time zones as dictated by public policy (when we shift between EST/EDT). FXCM data is *all* in EST.

Reply:
"and in the Central Time Zone (Chicago Time) during the other eight months" 🤣

Reply:
thanks for the clarification. Just edited it